src/components/ResultsMap.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+	<GmapMap
```

Comment:
Not sure if this is just GitHub, but the tabbing is definitely over-done here.  Does it look like this locally as well?  If so, could you fix it so there are less spaces before each line?

Reply:
I believe this is an error with the Github interface. See the comment left further below that includes a picture of how it is currently spaced inside of my local code editor.